src/components/BusinessController/index.js

```diff
@@ -135,11 +135,17 @@ export const BusinessController = (props) => {
     }
   }, [])
 
+  const updateBusiness = async (businessId, updateParams = {}) => {
+    const { content: { result } } = await ordering.businesses(businessId).save(updateParams)
+    setBusinessObject(result)
+  }
+
```

Comment:
use try catch and an state to manage this fetch (loading, error and result)

Reply:
Added try-catch and modify a state to properly handle fetch